config/topic_finders/prepare-individual-uk-leaving-eu.yml.erb

```diff
@@ -0,0 +1,36 @@
+# Required config keys for processing the ERB
+# :finder_content_id
+# :timestamp
+# :topic_content_id
+# :topic_name
+# :topic_slug
+
+base_path: "/prepare-eu-exit-live-uk/<%= config[:topic_slug] %>"
+content_id: "<%= config[:finder_content_id] %>"
+title: "<%= config[:topic_name] %> – EU Exit guidance"
+description:
+name: "<%= config[:topic_name] %> – EU Exit guidance"
+locale: en
+public_updated_at: "<%= config[:timestamp] %>"
+publishing_app: rummager
+rendering_app: finder-frontend
+details:
+  beta: false
+  document_noun: publication
+  facets: []
+  filter:
+    all_part_of_taxonomy_tree:
```

Comment:
Is additional work required in `finder-frontend` in order for this filter to work? I remember it was necessary for faceting, but perhaps not for filtering.

Reply:
It works in exactly the same way as having a single taxon set as the `filter` and then one set as a facet (so like the `brexit` checkbox on the news and comms finder).